modules/apps/dynamic-data-mapping/dynamic-data-mapping-web/src/main/resources/META-INF/resources/js/ddm_form.js

```diff
@@ -1297,7 +1297,7 @@ AUI.add(
 							locale === defaultLocale ||
 							(localizationMap[defaultLocale] !== undefined &&
 								value !== localizationMap[defaultLocale]) ||
-							localizationMap[locale]
+							localizationMap[locale] !== undefined
```

Comment:
@NemethNorbert do you think it is necessary to check that `localizationMap[locale] !== null`?

Reply:
Hey @carolmariaabb , 
Yes that check allows to update the locale in the localisationMap even if its current value an empty string